.gitignore

```diff
@@ -22,6 +22,7 @@ pnpm-debug.log*
 # environment variables
 .env
 .env.production
+src/env.d.ts
```

Comment:
Why is this being added to `.gitignore` as well as modified in this PR? The build process doesn't seem to modify it, so I don't think we need to ignore it?